### DISCUSSION TITLE
User agent for Floorp

### DISCUSSION THREAD

**Discussion by lunik1** [Floorp-Projects/Floorp#1512] [Feature Request]
Some websites like OpenRSS require clients to identify themselves uniquely via their user agent in their terms (see https://openrss.org/guides/developers-guide-to-open-rss-feeds#user-agents). Currently Floorp just seems to inherit it from Firefox (something like `Mozilla/5.0 (X11; Linux x86_64; rv:128.0) Gecko/20100101
Firefox/128.0`). Could `Floorp` be included in the user agent string?

**Comment by CutterKnife** [comment]
Floorp uses the Firefox user agent because browsers with fewer users may cause fingerprinting if they use a user agent that uniquely identifies it.
